tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -235,6 +274,17 @@ <h3>No profile data was found.</h3>
         type: Array,
         observer: '_activeHostsChanged'
       },
+      _inColab: {
+        type: Boolean,
+        value: () => !!(window.TENSORBOARD_ENV || {}).IN_COLAB,
+        readOnly: true,
+      },
+      /** @type {TopLevelState} */
+      _topLevelState: {
+        type: String,
+        computed: '_computeTopLevelState(_inColab, _dataNotFound, progress)',
+        readOnly: true,
```

Comment:
computed is readOnly. https://polymer-library.polymer-project.org/1.0/docs/devguide/properties

Reply:
Good to know; thanks.